src/gui/guiFormSpecMenu.cpp

```diff
@@ -1959,7 +1961,7 @@ void GUIFormSpecMenu::regenerateGui(v2u32 screensize)
 	parserData mydata;
 
 	//preserve tables
-	for (auto &m_table : m_tables) {
+	for (auto& m_table : m_tables) {
```

Comment:
Formatter did a bad job here. Please undo. Same for L1979 and L1991 in this file.